test/e2e/login.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect, Page } from "@playwright/test";
+import { LoginPage } from "../pages/LoginPage";
+import { base_url, invalidEmail, validEmail, validPassword } from "../helper-config";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(base_url);
+});
+
+test("login with valid credentials", async ({ page }) => {
+  const loginPage = new LoginPage({ page });
+  await loginPage.navigateToLoginPage();
```

Comment:
IMHO, this should happen for every login steps. So better put this in the beforeEach(Given) step.
```suggestion
  await loginPage.navigateToLoginPage();
```